packages/apps/src/Apps.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright 2017-2019 @polkadot/apps authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { BareProps } from '@polkadot/ui-app/types';
+
+import React from 'react';
+import styled from 'styled-components';
+import { classes } from '@polkadot/ui-app/util';
+import Signer from '@polkadot/ui-signer/index';
+import settings from '@polkadot/ui-settings';
+
+import { hot } from 'react-hot-loader/root';
+
+import Connecting from './Connecting';
+import Content from './Content';
+import SideBar from './SideBar';
+
+type Props = BareProps & {};
+
+const Wrapper = styled.div`
+  align-items: stretch;
+  box-sizing: border-box;
+  display: flex;
+  min-height: 100vh;
+`;
+
+function Apps (props: Props) {
+  return (
+    <Wrapper className={classes(`theme--${settings.uiTheme}`)}>
+      <SideBar />
+      <Signer>
+        <Content />
+      </Signer>
+      <Connecting />
+    </Wrapper>
+  );
+}
+
+export default hot(Apps);
```

Comment:
When we have `env.NODE_ENV == 'production'` we probably don't want to be doing this. In this case, rather just return without the HOC wrapper.

Reply:
The react-hot-loader package is already taking care of that, but I can add it as an additional security https://github.com/gaearon/react-hot-loader/blob/94d1379c5b1387c387362c5e76f77fee4a102088/index.js#L5

Reply:
It's the reason why I chose using the package instead of wrapping it manually like this https://github.com/polkadot-js/apps/blob/c6268643d48ba8f350faaa241eb9d5180f36345a/packages/apps/src/index.tsx#L75

Reply:
Ahhh. Black magic. then starting to wonder why we did it explicitly in the Ethereum days - could just have been for sanity.